lib/hero-image-header/index.js

```diff
@@ -1,10 +1,20 @@
 import React, { Component, PropTypes } from 'react';
+var Scroll = require('react-scroll');
```

Comment:
I guess we should have checked that - basically we needed a way of scrolling smoothly. We can look for a different module if necessary.